cognite_toolkit/cdf_tk/load.py

```diff
@@ -855,12 +863,46 @@ def create(self, items: Sequence[T_Resource], drop: bool, filepath: Path) -> T_R
                         if item.external_id == ext_id:
                             items.remove(item)
                 try:
-                    return ExtractionPipelineList(self.client.extraction_pipelines.create(items))
+                    extractionPipelineList = ExtractionPipelineList(self.client.extraction_pipelines.create(items))
                 except Exception as e:
                     print(f"[bold red]ERROR:[/] Failed to create extraction pipelines.\n{e}")
                     self.ToolGlobals.failed = True
                     return None
-            return None
+
+        file_name = filepath.stem.split(".", 2)[1]
+        config_file_stem = f"{file_name}.config"
+        config_file = next(
+            (file for file in Path(filepath.parent).iterdir() if file.is_file() and config_file_stem in file.name),
+            None,
+        )
+
+        if not config_file.exists():
+            print(
+                f"  [bold yellow]WARNING:[/] no config file for extraction pipeline found. Expected to find {config_file_stem} in same folder as {file_name}"
+            )
+            return extractionPipelineList
+
+        resources = load_yaml_inject_variables(config_file, {})
+        resources = [resources] if isinstance(resources, dict) else resources
+
+        for resource in resources:
+            extractionPipelineConfig = ExtractionPipelineConfig.load(
+                {
+                    "externalId": resource.get("externalId"),
+                    "description": resource.get("description"),
+                    "config": yaml.dump(resource.get("config", ""), indent=4)
+                    if config_file.suffix == ".yaml"
+                    else str(resource.get("config", "")),
+                }
+            )
+            try:
+                self.client.extraction_pipelines.config.create(extractionPipelineConfig)
+
+            except Exception as e:
+                print(f"[bold red]ERROR:[/] Failed to create extraction pipeline config.\n{e}")
+                self.ToolGlobals.failed = True
```

Comment:
I would expect this to look like this: (Assuming we iterate over all extraction_pipelines to find their configs)
```suggestion
        extraction_pipeline_config = ExtractionPipelineConfig(
            external_id=extraction_pipeline.external_id,
            config=config_file.read_text(),
        )
        try:
            self.client.extraction_pipelines.config.create(extraction_pipeline_config)

        except Exception as e:
            print(f"[bold red]ERROR:[/] Failed to create extraction pipeline config.\n{e}")
            self.ToolGlobals.failed = True
```

Reply:
We shouldn't assume they always have a config, it is acceptable to create e.p's for other purposes too.